source/standards/optimise-frontend-perf.html.md.erb

```diff
@@ -96,7 +96,7 @@ You can find out more about improving your website’s frontend performance by r
 The Service Manual has more suggestions about [how you can test frontend performance][].
 
 [frontend performance]: https://www.gov.uk/service-manual/technology/how-to-test-frontend-performance
-[minification]: https://minifycode.com/
+[minification]: https://web.dev/reduce-network-payloads-using-text-compression/
```

Comment:
We've updated this URL but removed the link from minification on line 21. Is that intentional? Is this link referred to from anywhere else?

Reply:
This was a mistake, I've added the link back